### PR TITLE
clDNN RESIZE_BILINEAR

### PR DESCRIPTION
**Pull request by huningxin**
fix https://github.com/intel/webml-polyfill/issues/336

### CODE REVIEW

**Comment by huningxin** [comment]
## PR Submission Checking Summary

### Baseline Information:
Chromium version: 4d20cb1
Webml-polyfill version: 5b85ade
### PR Submission Proposal: OK
### PR Submission Message:
Total Test Cases: 1698
Pass to Fail: 0
Fail to Pass: 10

File: services/ml/model_impl_cl_dnn.cc [hunk 1]
```diff
@@ -1629,4 +1633,99 @@ int32_t ModelImplClDnn::CldnnAddFullyConnected(
...
+                << std::string(LATE(cldnn_get_last_error_message)());
+    return mojom::OP_FAILED;
+  }
+  cldnn_custom_gpu_primitive_desc custom_desc = {.type = type_id};
+
+  // Setup inputs.
+  std::vector<std::string> input_ids(1);
+  std::vector<cldnn_primitive_id> input_ids_array(1);
+  input_ids[0] = base::NumberToString(inputs[0]);
+  input_ids_array[0] = input_ids[0].c_str();
```
[the remark this answers is not in the document]
Comment (huningxin) [reply services/ml/model_impl_cl_dnn.cc]: You may look at the first commit. The latest code uses this pattern. please take another look.

[the remark this answers is not in the document]
Comment (fujunwei) [reply services/ml/model_impl_cl_dnn.cc]: Get it, thanks.

**Comment by fujunwei** [comment]
LGTM.

**Comment by huningxin** [comment]
Thanks for review!

**Pull request merged**